config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  post 'notifications/notify' => 'notifications#notify'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.